hvac_services/demo/hr_employee.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="hr.employee_admin" model="hr.employee" forcecreate="1">
```

Comment:
If it happens to be created again with forcecreate, I bet you'd need some required fields defined here, like the name, or it would fail